src/components/form/AmountInput/AmountInput.tsx

```diff
@@ -77,9 +76,11 @@ export const AmountInput = forwardRef<HTMLDivElement, AmountInputProps>(
     const newPlaceholder = placeholder ?? definedDefaultPlaceholder(currency, translate)
 
     return (
-      <StyledTextInput
+      <TextInput
         ref={ref}
-        $displayErrorText={displayErrorText}
+        InputProps={{
+          style: !displayErrorText ? { marginBottom: '0 !important' } : {},
```

Comment:
is !important mandatory?

Reply:
well, need to find an implem but have tried to be iso in the meantime

Will look

Reply:
that's been unessesary recently, since the `TextInput` does not use margin but gap internally now

so the spacing is automatically not present if there is no error displayed

Will remove!

Reply:
Done